src/modules/controller.js

```diff
@@ -0,0 +1,19 @@
+import { displayModal, modal } from './modal.js';
+
+const openModal = () => {
+  document.addEventListener('click', async (e) => {
+    if (e.target.id === 'comment-btn') {
+      displayModal(e.target);
+    }
```

Comment:
- [ ] Please kindly add `await` on this asynchronous function

 ``
                await displayModal(e.target)
  ``